.github/workflows/deploy.yml

```
@@ -0,0 +1,55 @@
+# name: Deploy to Staging
```

actionlint

workflow is empty

(syntax-check)